.ci.yaml

```diff
@@ -80,9 +80,9 @@ platform_properties:
     properties:
       dependencies: >-
         [
-          {"dependency": "xcode"},
-          {"dependency": "gems"},
-          {"dependency": "apple_signing"}
+          {"dependency": "xcode", "version": "13a233"},
+          {"dependency": "gems", "version": "v3.3.14"},
+          {"dependency": "apple_signing", "version": "none"}
```

Comment:
Why would this be none?

Reply:
Yeah so I spoke with Drew and there is no version associated with this. And in the recipe there is no check on the version for this dependency so rather than leave it blank I just gave it None. I can replace it with something else if it makes more sense.

Reply:
Must be a well built tool to not need versions :-)